tools/apigee-x-trial-provision/README.md

```diff
@@ -127,3 +127,180 @@ the `CERTIFICATES` environment variable:
 |`generated`|Auto-Generated Self-Signed Certs|$RUNTIME_HOST_ALIAS or else $ORG-eval.apigee.net|
 |`supplied`|User-supplied in `RUNTIME_SSL_CERT` and `RUNTIME_SSL_KEY`|$RUNTIME_HOST_ALIAS or else $ORG-eval.apigee.net|
 <!-- markdownlint-enable MD013 -->
+
+## Apigee X instance with Custom Networking Provisioning
+<!-- markdownlint-disable MD013 -->
+
+## VPC Network Planning
```

Comment:
should this be `###`?

Reply:
What is the purpose of this section? Explain CIDR ranges?

Reply:
### --> fixed


The section is expanded with comments and doc links.